client/elements/menus/more-par-menu.html

```diff
@@ -145,6 +203,12 @@
                   return this.citeData = citefinalplugin + "Retrieved from https://suttacentral.net/" + inputData.url + " on " + d + ".";
               }
           }
-      });
+
+          _getAPIEndpoint(item) {
+              return `api/parallels/${item.uid}`;
```

Comment:
We should use `<iron-meta>` here to get the API url. See how it's done in other elements.